theano/gpuarray/opt.py

```diff
@@ -389,17 +389,18 @@ def apply(self, fgraph):
             if new_o.type != o.type:
                 assert isinstance(o.type, tensor.TensorType)
                 assert isinstance(new_o.type, GpuArrayType)
-                assert new_o.owner
```

Comment:
Yeah, i accidentally stumbled across this when i was trying to replace `host_from_gpu` calls with transfers. The tests inside `test_basic_ops` fail. These tests mainly, 
`test_basic_ops.py:test_transfer_strided`
`test_basic_ops.py:test_transfer_cpu_gpu`